TOOLS/lua/autocrop.lua

```diff
@@ -26,15 +31,15 @@ into mpv.conf
 List of available parameters (For default values, see <local options>)：
 
 auto: bool - Whether to automatically apply crop at the start of playback.
-    If you don't want crop automatically, set it to false or
+    If you don't want to crop automatically, set it to false or
     add "script-opts-append=autocrop-auto=no" into mpv.conf.
 
-auto_delay: seconds - Delay before starting crop.
-    this only effective when playback time is less than this.
+auto_delay: seconds - Delay before starting crop in auto mode.
     You can try to increase this value to avoid dark scene or fade in at beginning.
+    The auto delay is only effective when playback time is less than this.
```

Comment:
Maybe just rephrase it as "Automatic cropping will not occur if the value is larger than the remaining playback time."